core/app/assets/stylesheets/refinery/sections/_menu_item_icons.scss

```diff
@@ -0,0 +1,9 @@
+.site-name {@include icon('bars', $plugin_icon_colour, $margin: 1.2em)}
+
+.refinery_images_icon {@include icon('photo', $plugin_icon_colour, $margin: 1.2em)}
```

Comment:
@anitagraham My only concern is to not have a new CSS file in each extension to define an icon. I f not we will have to update the form/engine generator to add this new file.

I prefer the alternative to pre-create all class for each icon in core.

@stefanspicer it should be.